src/features/autothread.ts

```diff
@@ -60,9 +61,14 @@ const autoThread: ChannelHandlers = {
         return;
       }
     }
-    msg.startThread({
+    const newThread = await msg.startThread({
       name: `${msg.author.username} – ${format(new Date(), "HH-mm MMM d")}`,
     });
+    const message = await newThread.send(
+      `React to someone with ✅ to mark their response as the accepted answer and close this thread. If someone has been really helpful, give them a shoutout in <#${CHANNELS.thanks}>!`,
+    );
+    await sleep(30);
```

Comment:
Maybe instead of arbitrary waiting for 30 seconds, it would be more UX friendly to show that instruction after somebody (a non-author) will post a message in the thread (or start counting 30 seconds after such post appears)? I think, ideally bot could recognize whether author replied with for example: "thanks", "it helped", "it worked" etc. and immediately notify the author about marking possibility, but that may require some difficult AI stuff. :)

Reply:
Or we could simply send it after the first message in the thread